<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>591</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Consultas usuario Esporádico</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="result_frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>741</width>
     <height>351</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>621</width>
      <height>331</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::NoFocus</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>120</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Codigo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cedula</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nombre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Direccion</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Telefono</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>710</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QFrame" name="info_frame">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>691</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QRadioButton" name="radioButtonC1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>461</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Selecionar los empleado que son técnocis en finanzas.</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonC2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccionar los empleados que son asesores comerciales.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generarConsulta_btn_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Generar consulta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="limpiar_btn_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Consultas:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
